cetmix_tower_server/views/cx_tower_server_view.xml

```diff
@@ -360,7 +372,7 @@
                 <field name="name" />
                 <field name="reference" />
                 <field name="status" />
-                <field name="os_id" />
+                <field name="os_id" groups="cetmix_tower_server.group_manager" />
```

Comment:
Question: why do we need to add `group` attribute to the view definition?

Reply:
I agree, I will fix it